server/tests/controllers/quotes_controller.test.ts

```diff
@@ -61,4 +61,40 @@ describe("QuotesController", () => {
         });
     });
   });
+
+  describe("group", async () => {
+    let fakeQuotes = [
+      [
+        { quote: "Fake", author: "Fake Author" },
+        { quote: "Fake 2", author: "Fake Author 2" }
+      ],
+      [
+        { quote: "Fake", author: "Fake Author" },
+        { quote: "Fake 2", author: "Fake Author 2" }
+      ],
+      [
+        { quote: "Fake", author: "Fake Author" },
+        { quote: "Fake 2", author: "Fake Author 2" }
+      ],
+      [
+        { quote: "Fake", author: "Fake Author" },
+        { quote: "Fake 2", author: "Fake Author 2" }
+      ]
+    ];
+
+    beforeEach(() => {
+      // Stub the output of Quotes to provide a specific quote grouping
+      sinonSandbox.stub(Quotes, "getGroup").returns(fakeQuotes);
+    });
+
+    it("returns 200 and a collection of quotes", async () => {
+      await request(server)
+        .get("/api/quotes/group")
+        .expect("Content-Type", /json/)
+        .expect(200)
+        .then(response => {
+          expect(response.body.length).to.deep.equal(fakeQuotes.length);
```

Comment:
```suggestion
          expect(response.body).to.deep.equal(fakeQuotes);
```

It would be more precise to deep equal that the output is the same.

Reply:
But here is where my understanding breaks down. What does response.body actually mean?
Are we testing the function with the fakequotes data or with the real quotes data, or one to the other?